client/src/components/RegisterModal.vue

```diff
@@ -0,0 +1,263 @@
+<script setup lang="ts">
+import { ref, defineEmits } from 'vue'
+import { createVuetify } from 'vuetify'
+import { VDateInput } from 'vuetify/labs/VDateInput'
```

Comment:
Don't use components from Vuetify labs -- they aren't meant for production use (see https://vuetifyjs.com/en/labs/introduction/#what-is-labs). I think a standard input element with ``type="date"`` should be sufficient.